dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -84,6 +101,72 @@ public static void main(String[] arguments) throws IOException, ClassNotFoundExc
     System.out.println(stringBuilder.toString());
   }
 
+  /**
+   * Parses arguments to get list of jar file paths.
+   *
+   * @param arguments command-line arguments
+   * @return list of the absolute paths to jar files
+   * @throws RepositoryException when there is a problem in resolving the Maven coordinate to jar
+   */
+  static List<Path> parseArguments(String[] arguments) throws RepositoryException {
```

Comment:
@VisibleForTesting if need be, otherwise private

Reply:
Missed this comment. I'll  add the annotation in next PR.